core/src/com/biglybt/plugin/net/buddy/BuddyPlugin.java

```diff
@@ -288,11 +289,10 @@
 					});
 	
 			os_param.setVisible( SUPPORT_ONLINE_STATUS  ); // If we add this then use proper message texts in the STATUS_STRINGS
-	
-			
-			HyperlinkParameter profile_link = 
+
+			HyperlinkParameter profile_link =
 					config.addHyperlinkParameter2( 
-						"azbuddy.profile.info", MessageText.getString( "azbuddy.profile.info.url" ));

```

Comment:
@parg @TuxPaper 
Is this a safe change? 
I can't determine if the different plugins are supposed to use a spesific API or not. 
Is there a difference between plugins shipped in "core" module and the external ones?

Reply:
If there is a strict API then a forward compatible change would probably be
`Constants.URL_WIKI + "w/Friends#public-profile"`

Reply:
Not sure if your changes handle this - these two keys

azbuddy.send.friend.key.msg
azbuddy.add.friend.key.msg

are sent via chat unexpanded to explain what's going on to the recipient. On the receiver's side they are expanded and shown to the user (in their local language of course). Both of these message texts contain a wiki url expansion:  {azbuddy.classic.link.url}

As for plugin stuff, they are supposed to use the plugin interface's LocaleUtilities interface and not MessageText. However, for built-in ones the need to stick to plugin-interface stuff is widely flouted. Indeed, for a lot of non-built-in ones (particularly ones written by ourselves), non-PI stuff is often used. Given that we hardly have a vibrant community of plugin developers I really wouldn't consider formalizing plugin access to things, just try and ensure that things don't get broken :)

Reply:
Thank you for elaborating. 
I believe the keys you mention won't pose any issue as it's up to the receiver to translate them. 

I'm moving the expansion keys needed for this plugin to the initialization block - as they are spesific to this plugin only.
Like you're saying, as long as it's a built in plugin then there's no need to be too worried about strictly conforming to the plugin-api.